File: PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthResultInfo.java
package com.braintreepayments.api.paypal;

import androidx.annotation.NonNull;

import com.braintreepayments.api.BrowserSwitchFinalResult;

/**
 * Details of a {@link PayPalPaymentAuthResult.Success}
 */
public class PayPalPaymentAuthResultInfo {

    private final BrowserSwitchFinalResult.Success browserSwitchSuccess;

    PayPalPaymentAuthResultInfo(@NonNull BrowserSwitchFinalResult.Success browserSwitchSuccess) {
        this.browserSwitchSuccess = browserSwitchSuccess;
    }

    @NonNull
    BrowserSwitchFinalResult.Success getBrowserSwitchResult() {
        return browserSwitchSuccess;
    }
}

File: PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthResultInfo.kt
package com.braintreepayments.api.paypal

import androidx.annotation.RestrictTo
import com.braintreepayments.api.BrowserSwitchFinalResult

/**
 * Details of a [PayPalPaymentAuthResult.Success]
 */
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
data class PayPalPaymentAuthResultInfo internal constructor(
    val browserSwitchSuccess: BrowserSwitchFinalResult.Success
)
